packages/forma-36-website/src/pages/components/grid/index.mdx

```diff
@@ -6,23 +6,153 @@ github: 'https://github.com/contentful/forma-36/tree/master/packages/forma-36-re
 storybook: 'https://f36-storybook.contentful.com/?path=/story/components-grid--default'
 ---
 
-Detailed documentation will be provided soon. For now, please have a look into the storybook for implementation details.
+import tokens from '@contentful/forma-36-tokens';
```

Comment:
@mshaaban0 Why do we need this import?

Reply:
@burakukula Ah, good spot, removed! 
I tried to use it but didn't work, forgot to remove it!